app/assets/stylesheets/base/typography.scss

```diff
@@ -96,27 +105,79 @@ body {
   h5,
   h6 {
     font-family: var(--ff-sans-serif);
-    margin: var(--content-rythm) 0;
-    margin-top: calc(var(--content-rythm) * 2);
+    margin: calc(var(--content-rhythm) * 2) 0 calc(var(--content-rhythm) / 2);
```

Comment:
The current approach is a 2:1 ratio for the margin from top:bottom.

Another approach is a 3:1 ratio to make rhythm and contrast even clearer; what I proposed in the code is basically 4:1 which might be too drastic.